scripts/auth.js

```diff
@@ -0,0 +1,25 @@
+//<---------------------------------------------------------------------------------------->
+//THIS IS JUST THE GOOGLE SIGN IN CODE
+function onSignIn(googleUser) {
+    // Useful data for your client-side scripts:
+    var profile = googleUser.getBasicProfile();
+    console.log("ID: " + profile.getId()); // Don't send this directly to your server!
+    console.log('Full Name: ' + profile.getName());
+    console.log('Given Name: ' + profile.getGivenName());
+    console.log('Family Name: ' + profile.getFamilyName());
+    console.log("Image URL: " + profile.getImageUrl());
+    console.log("Email: " + profile.getEmail());
```

Comment:
i think we should remove any console.log statements that we don't need for sign in functionality

Reply:
> i think we should remove any console.log statements that we don't need for sign in functionality

@lalaAdasia00 If we are keeping log statements around for the prototype (which is okay), it would be best to add a // TODO to remove it in the future.

Something like `// TODO(ldap or bug number here) Remove this`